runtime/Cargo.toml

```diff
@@ -8,4 +8,4 @@ edition = "2021"
 [dependencies]
 counters = { git = "https://github.com/oxidecomputer/hubris" }
 userlib = {git = "https://github.com/oxidecomputer/hubris"}
-zerocopy = "0.6.1"
+zerocopy = { version = "0.8.25", features = ["derive"] }
```

Comment:
Nit:

this is not how the zerocopy folks currently recommend getting the derive macros. From the crate docs:

> However, you may experience better compile times if you instead directly depend on both zerocopy and zerocopy-derive in your Cargo.toml, since doing so will allow Rust to compile these crates in parallel.

Reply:
Turns out `idol-runtime` doesn't actually use the derives, just the traits. So we can just drop the feature here. However, the codegen will need to be updated if we want the rest of Hubris to derive on the two of them separately.